portfolio/src/main/java/com/google/sps/servlets/FactsServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/facts")
+public class FactsServlet extends HttpServlet {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 8931978960646374589L;
+
+    @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> facts = new ArrayList<String>();
```

Comment:
Nit: Since you declared the type on the left hand size(ArrayList<String>) you don't have to do so on the right hand side. You can make use of the diamond shorthand syntax: ArrayList<>().